app/jobs/webhook_transmit_job.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+# License: AGPL-3.0-or-later WITH WTO-AP-3.0-or-later
+# Full license explanation at https://github.com/houdiniproject/houdini/blob/master/LICENSE
+class WebhookTransmitJob < ApplicationJob
+  queue_as :default
+
+  def perform(webhook, payload)
+    webhook.transmit(payload)
+  end
```

Comment:
Not sure if passing the webhook to the job is okay, I can pass the ObjectEventHookConfig id and do a `find_by_id` here

Reply:
See below. The quick answer is 'not really'.